hugegraph-test/src/main/java/com/baidu/hugegraph/api/UserApiTest.java

```diff
@@ -168,14 +169,12 @@ protected void createUser(String name) {
         this.client().post(path, user);
     }
 
-    protected List<Map<String, Object>> getList() {
-        Response r = this.client().get(path,
-                                       ImmutableMap.of("limit", -1));
+    protected List<Map<String, Object>> listUsers() {
+        Response r = this.client().get(path, NO_LIMIT);
         String result = assertResponseStatus(200, r);
-        Map<String, List<Map<String, Object>>> resultMap = JsonUtil.fromJson(
-                result,
-                new TypeReference<Map<String, List<Map<String, Object>>>>() {
-                });
+        Map<String, List<Map<String, Object>>> resultMap =
+                JsonUtil.fromJson(result, new TypeReference<Map<String,
+                                  List<Map<String, Object>>>>() {});
```

Comment:
return JsonUtil.fromJson(result, type).get("users");

Reply:
it need to cast

Reply:
update alignment